scripts/ipfs-pi-stream/install

```diff
@@ -24,3 +24,14 @@ sudo systemctl enable process-stream
 
 # Add hourly job to clear out old data
 echo "41 * * * * $USER /usr/local/bin/ipfs repo gc" | sudo tee --append /etc/crontab
+
+# Install the ipfs video player
+mkdir "$BASE_DIR/tmp"
+git clone https://github.com/tomeshnet/ipfs-live-streaming.git "$BASE_DIR/tmp/live-streaming"
```

Comment:
We better call this directory `ipfs-live-streaming`. In general, I prefer to avoid unnecessary name mapping in code.